tests/session_tests.py

```
@@ -5,34 +5,50 @@
-                "-u",
-                os.path.join(
-                    os.path.dirname(os.path.abspath(__file__)),
-                    "utils",
-                    "session_helper.py",
-                ),
-            ]
-            + list(args)
-        )
-        .decode()
-        .replace("\r", "")
+def _make_session_shell(start_time: float, disable_warning: bool) -> fos.Session:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)